README.md

```diff
@@ -1,4 +1,4 @@
-# GOV.UK Forms Admin [![Ruby on Rails CI](https://github.com/alphagov/forms-admin/actions/workflows/rubyonrails.yml/badge.svg)](https://github.com/alphagov/forms-admin/actions/workflows/rubyonrails.yml) [![Deploy to GOV.UK PaaS](https://github.com/alphagov/forms-admin/actions/workflows/deploy.yml/badge.svg)](https://github.com/alphagov/forms-admin/actions/workflows/deploy.yml)
+# GOV.UK Forms Admin [![Ruby on Rails CI](https://github.com/alphagov/forms-admin/actions/workflows/test.yml/badge.svg)](https://github.com/alphagov/forms-admin/actions/workflows/test.yml)
```

Comment:
This'll conflict with @lfdebrux's change here: https://github.com/alphagov/forms-admin/pull/553/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1
Should be easy enough to resolve, just a heads up to whichever of you is slower to merge 😄

Reply:
he beat me to it. But I'm happy it's happened 🥳